satpy/etc/writers/awips_tiled.yaml

```diff
@@ -0,0 +1,1125 @@
+      viirs_adaptive_dnb:
+        name: adaptive_dnb
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: Adaptive DNB
+      viirs_dynamic_dnb:
+        name: dynamic_dnb
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: Dynamic DNB
+      viirs_hncc_dnb:
+        name: hncc_dnb
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: HNCC DNB
+      viirs_ifog:
+        name: ssec_fog
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: Fog
+      viirs_m01:
+        name: M01
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.41 um
+      viirs_m02:
+        name: M02
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.45 um
+      viirs_m03:
+        name: M03
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.49 um
+      viirs_m04:
+        name: M04
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.56 um
+      viirs_m05:
+        name: M05
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.67 um
+      viirs_m06:
+        name: M06
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.75 um
+      viirs_m07:
+        name: M07
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 0.86 um
+      viirs_m08:
+        name: M08
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 1.24 um
+      viirs_m09:
+        name: M09
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 1.38 um
+      viirs_m10:
+        name: M10
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 1.61 um
+      viirs_m11:
+        name: M11
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 2.25 um
+      viirs_m12:
+        name: M12
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 3.70 um
+      viirs_m13:
+        name: M13
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 4.05 um
+      viirs_m14:
+        name: M14
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 8.6 um
+      viirs_m15:
+        name: M15
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 10.8 um
+      viirs_m16:
+        name: M16
+        var_name: data
+        attributes:
+          physical_element:
+            raw_value: 12.0 um
+
+      # VIIRS Corrected Reflectance
+      #    viirs_viirs_crefl01:
```

Comment:
Do we need these comments?

Reply:
They were here before and were probably used in Polar2Grid. I think they are for datasets/readers that haven't been ported to Satpy yet. :thinking: I'd like to keep them here for now.

Reply:
Ok